tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "strictNullChecks": true,
```

Comment:
I enabled the strictNullChecks, because it showed hidden bugs in the code base 😅 We should have an additional look at this ❗️